common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -7303,6 +7304,13 @@ public Map<String, String> subtree(String string) {
     return ret;
   }
 
+  public static boolean shouldComputeLineage(HiveConf conf) {
```

Comment:
Does it mean that we completely want to remove using POST_EXEC_HOOKS? It is a breaking change. Can we do this without a major release?

Reply:
Yes, @InvisibleProgrammer , it will be breaking change. I will create a doc for the same in hive-site website to help users.
 
My understanding is that next release should be major release as we have upgrade Hadoop version, Tez upgrade(not yet done but will be done before next release) and JDK17 (if possible) is planned.

Reply:
@Aggarwal-Raghav 
Are you planning document it here?
https://github.com/apache/hive-site